README.md

```diff
@@ -82,7 +82,7 @@ Regarding this package, we accept pull requests as well feature requests.
 Execute the following command from the root of the repository to run the tests:
 
 ```
-flutter test test/
+flutter test
```

Comment:
@peterwvj I know it's already been merged, but we added the `test/` directory for a good reason, around `UriReader`.
If `flutter test` now works without having to specify the directory, so much the better!